aioxmpp/muc/service.py

```diff
@@ -618,18 +633,28 @@ def _match_tracker(self, message):
         try:
             tracker = self._tracking_by_id[message.id_]
         except KeyError:
+            key = message.from_, _extract_one_pair(message.body)
```

Comment:
Are we sure we get the same pair as when registering the message? Should we not rather search whether any of the messages matches the one we selected when registering the message for tracking.